### ISSUE TITLE
Appointment update failure for items with empty Google description

### ISSUE THREAD

**Issue by SailingNut** [phw198/OutlookGoogleCalendarSync#1787]
I am seeing sync failures since the most recent update. This reproduces itself on every sync. I am running version 2.10.3.0
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/14079291/OGcalsync.log)

> Checklist:-
> * [x] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [x] Reproduce the issue
> * [x] Complete the information below
> * [x] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v1.2.3.4  
**Edition**: Installed / Portable  
**Sync Direction**: G->O / O->G / O<->G

**Bug Description**  

**Steps to Reproduce the Issue**  
1. Go to...
1. Click on...

**Any other information**  
Any additional information that may help troubleshoot the issue.  
:paperclip: Attach your `OGcalsync.log` file.
```
2024-01-28 22:22:51,064 ERROR  11 OutlookGoogleCalendarSync.Console [315] -  Appointment update failed.
2024-01-28 22:22:51,065 ERROR  11 OutlookGoogleCalendarSync.Console [315] -  Value cannot be null.
2024-01-28 22:22:51,065 ERROR  11 OutlookGoogleCalendarSync.Console [315] -  Parameter name: input
2024-01-28 22:22:51,079 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [28] -  System.ArgumentNullException: Value cannot be null.
Parameter name: input
2024-01-28 22:22:51,080 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [21] -  Code: 0x80004003,-2147467261; Location: Replace() in <Unknown File>
2024-01-28 22:22:51,080 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [65] -     at System.Text.RegularExpressions.Regex.Replace(String input, String replacement)
   at System.Text.RegularExpressions.Regex.Replace(String input, String pattern, String replacement)
   at OutlookGoogleCalendarSync.OutlookOgcs.Calendar.UpdateCalendarEntry(AppointmentItem& ai, Event ev, Int32& itemModified, Boolean forceCompare) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\OutlookOgcs\OutlookCalendar.cs:line 606
   at OutlookGoogleCalendarSync.OutlookOgcs.Calendar.UpdateCalendarEntries(Dictionary`2 entriesToBeCompared, Int32& entriesUpdated) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\OutlookOgcs\OutlookCalendar.cs:line 435
```

**Comment by galexmay** [comment]
Same here
Version	2.10.3.0
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/14080364/OGcalsync.log)

**Comment by Katanasnk** [comment]
Same here
Version 2.10.3.0

[OGcalsync.log.2024-01-28.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/14090579/OGcalsync.log.2024-01-28.txt)

**Comment by Spacekat21** [comment]
This is happening 30 times per day for me.  I'm at wit's end.

[OGcalsync.log.2024-01-29.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/14102361/OGcalsync.log.2024-01-29.txt)

**Comment by phw198** [comment]
Duplicate of #1776, which [has workaround](https://github.com/phw198/OutlookGoogleCalendarSync/issues/1776#issuecomment-1915657910).

**Comment by SailingNut** [comment]
I have disabled "Include Meet conferences" in the settings for all 3 of my sync profiles and the problem still exists.

**Comment by phw198** [comment]
Hi @SailingNut, you're right, the workaround only worked for Outlook to Google syncs so doesn't fix the error you were getting during Google to Outlook. 

However, the fix posted on #1776 should now provide a resolution for all who have posted here.
